eslint.config.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
     "project": "./tsconfig.json"
   },
   rules: {
+    "semi": "error",
```

Comment:
Shouln't we use rule `@typescript-eslint/semi` instead?

Reply:
Yeah, you are right. Updated.